Translations/Common/Translators.html

```diff
@@ -93,6 +93,7 @@ <h2>日本語</h2>
 		<h2>한국어</h2>
 		<span>JiHoon <a href="mailto:jihoongks@gmail.com">jihoongks@gmail.com</a></span>
 		<span>Lee, Kyutae <a href="https://github.com/gorisanson">https://github.com/gorisanson</a></span>
+		<span>Youngjin Mo <a href="https://github.com/youngjinmo">https://github.com/youngjinmo</a></span>
```

Comment:
This one is for the app translation, not the web translation :)

Reply:
Oh, I missed. I will fix it and PR again.